app/operations/shopify/product_operations.rb

```diff
@@ -0,0 +1,27 @@
+module Shopify
+  class ProductOperations
+    class << self
+      def create(spree_product:)
+        service = Shopify::ProductExporter.new(spree_product: spree_product)
+        service.perform
+      end
+
+      def update(spree_product:)
```

Comment:
Yes it's just a regular argument, but I felt like this would remove the confusion in either having to pass the spree_product or the shopify_product.